va/va_dec_jpeg.h

```diff
@@ -70,6 +70,10 @@ typedef struct _VAPictureParameterBufferJPEGBaseline {
     }                   components[255];
     /** \brief Number of components in frame (Nf). */
     unsigned char       num_components;
+    /** \brief Input color space 0: YUV, 1: RGB, 2: BGR, others: reserved */
+    unsigned char       color_space;
+    /** \brief Set to VA_ROTATION_* for a single rotation angle reported by VAConfigAttribDecJPEG. */
+    unsigned int 	rotation;
 } VAPictureParameterBufferJPEGBaseline;
```

Comment:
1.Jpeg itself doesn’t have notion of color space, the info is in user message. But for practical usage, you would need to have those info. Unlike other video codec, Jpeg can be coded in various formats but most common usage would be 420/422 or RGB which can be determined by output surface format. Without this info, we wouldn’t be able to convert to the correct output format, it’s not an Intel specific thing. 
2. Rotation is a nice feature that we can support it . Moving it out would make rotation generic and then we would need to have an attribute for each codec to indicate if rotation is supported or not. If we move it out to make it specific to Jpeg, that doesn’t make much sense